build/Helix/packages.config

```diff
@@ -0,0 +1,4 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<packages>
+  <package id="TAEF.Redist.Wlk" version="10.57.200918006-develop" targetFramework="native" />
```

Comment:
this refers to the one I uploaded to ProjectReunion-Dependencies?

Reply:
Correct yes. 
From pipeline running:
Added package 'TAEF.Redist.Wlk.10.57.200918006-develop' to folder 'D:\a\1\s\build\Helix\packages' from source 'https://pkgs.dev.azure.com/ms/ProjectReunion/_packaging/ProjectReunion-Dependencies/nuget/v3/index.json'